src/internal/ppsutil/util.go

```diff
@@ -120,7 +124,7 @@ func GetPipelineInfoAllowIncomplete(pachClient *client.APIClient, ptr *pps.Store
 	// ensure we are authorized to read the pipeline's spec commit, but don't propagate that back out
 	pachClient = pachClient.WithCtx(pachClient.Ctx())
 	pachClient.SetAuthToken(ptr.AuthToken)
-	if err := pachClient.GetFile(ptr.SpecCommit, ppsconsts.SpecFile, &buf); err != nil {
+	if err := pachClient.GetFile(ptr.OriginalSpecCommit, ppsconsts.SpecFile, &buf); err != nil {
```

Comment:
As a side note, I don't like the idea of attempting to proceed if we error when getting the spec file. I feel like this should be considered a critical error, and we shouldn't need to account for an incomplete pipeline info after attempting to get it. Not really suggesting any changes for this PR, just wanted to draw attention to it.